### PR TITLE
Custom configuration for Tenders.API

### PR DESCRIPTION
**Pull request by tatianaIljina**

### CODE REVIEW

File: API/Tenders.API.DAL.Postgres/ApiOptions.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    using global::Infrastructure.Common.Abstraction.Models;
+    using Microsoft.IdentityModel.Tokens;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    ///   <para>Api config service.</para>
+    /// </summary>
+    public class ApiOptions
+    {
+        private readonly string addresseesByDivisonTemplate = string.Empty;
```
Comment (Anny777) [inline API/Tenders.API.DAL.Postgres/ApiOptions.cs:18]: Convert to properties.

File: API/Tenders.API/Services/ApiLocalConfigurationProvider.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.Extensions.Configuration;
+    using Tenders.API.DAL.Postgres;
+
+    /// <inheritdoc/>
+    public class ApiLocalConfigurationProvider : ConfigurationProvider
```
Comment (Anny777) [inline API/Tenders.API/Services/ApiLocalConfigurationProvider.cs:15]: Rename on EntityConfigurationProvider  or DbConfigurationProvider

File: API/Tenders.API.DAL.Postgres/ApiOptions.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    using System;
+    using System.Collections.Generic;
+    using global::Infrastructure.Common.Abstraction.Models;
+    using Microsoft.IdentityModel.Tokens;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    ///   <para>Api config service.</para>
+    /// </summary>
+    public class ApiOptions
```
Comment (Anny777) [inline API/Tenders.API.DAL.Postgres/ApiOptions.cs:16]: Don`t rename class.

File: API/Tenders.API/Services/ApiOptions.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    using System;
+    using System.Collections.Generic;
+    using global::Infrastructure.Common.Abstraction.Models;
+    using Microsoft.IdentityModel.Tokens;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    ///   <para>Api config service.</para>
+    /// </summary>
+    public class ApiOptions
```
Comment (Anny777) [inline API/Tenders.API/Services/ApiOptions.cs:16]: Why this class same Tenders.API.DAL?

**Review by Anny777 — CHANGES_REQUESTED** [review]

File: API/Tenders.API.DAL.Postgres/Services/PostgresConfigService.cs [hunk 5]
```diff
@@ -4,19 +4,18 @@
 
 namespace Tenders.API.DAL.Postgres.Services
 {
+    using System;
     using Tenders.API.DAL.Postgres.Abstraction.Services;
-    using Tenders.Core.Abstraction.Services;
 
     /// <summary>Postgres Config Service.</summary>
-    /// <seealso cref="Tenders.API.DAL.Postgres.Abstraction.Services.IPostgresConfigService" />
     internal class PostgresConfigService : IPostgresConfigService
```
Comment (DamienVStone) [inline API/Tenders.API.DAL.Postgres/Services/PostgresConfigService.cs:11]: is it using anywhere?

**Pull request closed without merging**